content/en/docs/guides/server-graceful-stop.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Gracefull Shutdown
+description: >-
+  Explains how to gracefully stop/shutdown a gRPC server.
```

Comment:
Why complicate it with "stop/shutdown" here?  I'd recommend keeping one or the other but not both.  And should we say something else besides just repeating the title with "explains how" in front of it?  E.g.

"Explains how to gracefully shut down [or stop, if you prefer] a gRPC server to avoid causing RPC failures for connected clients."

(Note that as a verb, you should use "shut down" with a space.)

Reply:
Added. Perhaps we should with all guides